views/event-form.hbs

```diff
@@ -1,190 +1,218 @@
+    </tr>
+    <tr>
+      <td class="w-20 dark-text f4 b pa3 tr">{{localText.placeAddress}}</td>
+      <td class="w-100">
+          {{#if place.en.address}}
+            <input class="input-reset w-100 br2 ba b--gray" type="text" name="address_en" value="{{place.en.address}}">
+          {{else}}
+            <input class="input-reset w-100 br2 ba b--gray dib" type="text" name="address_en" placeholder="{{localText.english}}">
+          {{/if}}
+      </td>
+    </tr>
+    <tr>
+        <td class="w-20 dark-text f4 b pa3 tr">{{localText.eventsPlace}}</td>
+        <td class="w-100">
+          <select class="w-100" name="eventPlace">
+            <option value="">{{localText.selectPlace}}</option>
+            {{#each placesNameAndIds}}
+              <option value="{{this.id}}" {{checkDropDown ../event.place._id this.id}}>{{this.name}}</option>
+            {{/each}}
+          </select>
+        </td>
+    </tr>
+    <tr>
+      <td class="w-20 dark-text f4 b pb3 ph3 pt1 tr">{{localText.eventStart}}</td>
+      <td class="w-100">
+          {{#if event.startTime}}
+            <input class="input-reset w-100 br2 ba b--gray" type="datetime-local" name="startTime" value="{{event.startTime}}">
+          {{else}}
+            <input class="input-reset w-100 br2 ba b--gray dib" type="datetime-local" name="startTime">
+          {{/if}}
+      </td>
+    </tr>
+    <tr>
+      <td class="w-20 dark-text f4 b ph3 pb3 pt0 tr">{{localText.eventEnd}}</td>
+      <td class="w-100">
+          {{#if event.endTime}}
+            <input class="input-reset w-100 br2 ba b--gray" type="datetime-local" name="endTime" value="{{event.endTime}}">
+          {{else}}
+            <input class="input-reset w-100 br2 ba b--gray dib" type="datetime-local" name="endTime">
+          {{/if}}
+      </td>
+    </tr>
+    <tr>
+      <td class="w-20 dark-text f4 b ph3 pb3 pt0 tr">{{localText.eventsCost}}</td>
+      <td class="w-100">
+          {{#if event.cost}}
+            <input class="input-reset w-100 br2 ba b--gray" type="text" name="cost" value="{{event.cost}}">
+          {{else}}
+            <input class="input-reset w-100 br2 ba b--gray dib" type="text" name="cost">
+          {{/if}}
+      </td>
+    </tr>
+    <tr>
+      <td class="w-20 dark-text f4 b ph3 pb3 pt0 tr">{{localText.formImageUrl}}</td>
+      <td class="w-100">
+          {{#if event.imageUrl}}
+            <input class="input-reset w-100 br2 ba b--gray" type="text" name="imageUrl" value="{{event.imageUrl}}">
+          {{else}}
+            <input class="input-reset w-100 br2 ba b--gray dib" type="text" name="imageUrl">
+          {{/if}}
+      </td>
+    </tr>
+    <tr>
+      <td class="w-20 dark-text f4 b pa3 tr">{{localText.accessibilityOptions}}
+      </td>
+      <td><table><tr>
+          <td class="flex items-center mb2 ">
+            <input class="mr2" id="braille" type="checkbox" name="accessibility" value="Braille" {{checkOptionsValue place.accessibilityOptions "Braille"}}>
```

Comment:
all of the `{{checkOptionsValue place.accessibilityOptions`

should be `{{checkOptionsValue event.accessibilityOptions`